tests/Bedrock.Framework.Experimental.Tests/Http1Tests.cs

```diff
@@ -0,0 +1,48 @@
+using System.Buffers;
+using System.Net.Http;
+using System.Text;
+using Bedrock.Framework.Protocols;
+using Xunit;
+
+namespace Bedrock.Framework.Experimental.Tests
+{
+    public class Http1Tests
+    {
+        [Fact]
+        public void WriteHttp1MessageWithNoHost()
+        {
+            // arrange
+            var messageWriter = new Http1RequestMessageWriter("www.host.com", 8080);
+
+            var request = new HttpRequestMessage(HttpMethod.Get, "/api");
+            var output = new ArrayBufferWriter<byte>();
+
+            // act
+            messageWriter.WriteMessage(request, output);
+            var actual = Encoding.ASCII.GetString(output.WrittenSpan);
+
+            // assert
+            var expected = "GET /api HTTP/1.1\r\nHost: www.host.com:8080\r\n\r\n";
```

Comment:
I'd do a Assert.Contains check for "Host: www.host.com:8080" rather than the entire string. New lines may cause test issues.